src/components/media-player/media-player-track.js

```diff
@@ -166,7 +166,18 @@ function MediaPlayerTrack({
     // In this case the react-player automatically plays, disregarding the value passed to the "playing" prop.
     // Therefore we have to inform the parent player about the internal state damage and reload the whole player.
     if (trackPlayState.current === MEDIA_PLAY_STATE.pausing && progress.played === 0) {
-      playerRef.current.getInternalPlayer()?.pauseVideo();
+      playerRef.current.getInternalPlayer()?.pauseVideo?.();
```

Comment:
Apparently there is no `pauseVideo` method in the internal player when it's not a youtube source 😬

Reply:
Ahhh, of course!